packages/firestore/.npmignore

```diff
@@ -2,9 +2,10 @@
 /src
 test
 .idea
+.cache
```

Comment:
So I think this is meant to go in `.gitignore` at the top level, at least, that is what fixed my IDE wanting to add 1000 files to the commit after building.

Fun fact, I believe `.npmignore` files do not do anything in our repo anymore because you can either whitelist with the `files` field in `package.json` OR blacklist with `.npmignore` and if you use `files` it completely ignores `.npmignore`.  We should probably delete this and set up the package.json `files` whitelist to be a bit more granular (it will pack everything in /dist into the npm package right now - we actually do need `src` for `d.ts` files but we don't need `test`).

Reply:
But if we check them into git, then we save everyone a bunch of time pre-building their cache! Just kidding - I added them to .gitignore. I wonder why my IDE didn't ask me to commit those 1000 files.